src/style.css

```diff
@@ -1 +1,52 @@
-/* add styles here */
+body {
+  margin: 0 10;
+}
+
+.search-bar {
+  font-family: Avenir, sans-serif;
+  display: block;
+  width: 100%;
+  margin-top: 20px;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border: 1px solid #e1e7ed;
+  border-radius: 5px;
+  color: #3d4e61;
+  outline: none;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.search-bar--big {
+  height: 70px;
+  padding-left: 62px;
+  background-position: 26px center;
+}
+
+.search-bar--small {
+  height: 42px;
+  padding-left: 42px;
+  background-size: 11px;
+  background-position: 13px center;
+}
+
+.search-bar:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.search-bar:focus {
+  box-shadow: 0 4px 4px 0 #00000040;
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
```

Comment:
The `@font-face` rule specifies a font-weight of 'bold', which should be a numeric value like 700. Consider changing it to a numeric value for consistency.